### PR TITLE
Changeset v19.4.1-1 for correcting listnms.listdate

### PR DESCRIPTION
**Pull request by mchabito**
Hi @clarysabel , please review and merge. Thanks! cc @mcrimi

### CODE REVIEW

**Comment by clarysabel** [comment]
Hi @mchabito 
I did not include this PR in the 19.4.1 minor version because I did not see the ticket in the board and I did not merge it. 
We will have a 19.5 sprint to wrap up some details for the release, I push your ticket to 19.5 sprint
Could you please move your changeset to 19_5_0.xml and rename the ids? 
Everything else seems to be fine, I will merge it when the changes are done.

**Comment by mchabito** [comment]
Ok, no problem @Clarysabel Tovar ***@***.***>
Thanks !

**Review by darla-leafnode — APPROVED** [review]

**Pull request merged**
